CMakeLists.txt

```diff
@@ -25,13 +25,19 @@ endif()
 # Command line options
 #===============================================================================
 
-option(openmp   "Enable shared-memory parallelism with OpenMP"         ON)
-option(profile  "Compile with profiling flags"                         OFF)
-option(debug    "Compile with debug flags"                             OFF)
-option(optimize "Turn on all compiler optimization flags"              OFF)
-option(coverage "Compile with coverage analysis flags"                 OFF)
-option(dagmc    "Enable support for DAGMC (CAD) geometry"              OFF)
-option(libmesh  "Enable support for libMesh unstructured mesh tallies" OFF)
+option(OPENMC_ENABLE_PROFILE  "Enable shared-memory parallelism with OpenMP"         ON)
+option(OPENMC_ENABLE_COVERAGE "Compile with profiling flags"                         OFF)
+option(OPENMC_USE_OPENMP      "Compile with coverage analysis flags"                 OFF)
```

Comment:
Looks like something got messed up on these lines (descriptions don't match the variables)

Reply:
lol ya I know how that happened my bad 🤦‍♀️